README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
+| Deterministic randomness for randomised execution                                         | ✔️       | ✖️                                                           | C                                                                |
+| Deterministic randomness for randomised execution order by default                        | ✔️       | ✖️                                                           | C                                                                |
 | Deterministic randomness                                                                  | ✔️       | ✔️                                                           | ✔️                                                                |
 | Deterministic randomness by default                                                       | ✔️       | ✖️                                                           | ✖️                                                                |
-| Deterministic randomness instructions for rerunning tests                                 | 🚧      | ✔️                                                           | ?                                                                |
-| Hiding process arguments from tests                                                       | ✔️       | ✔️                                                           | ?                                                                |
+| Deterministic randomness instructions for rerunning tests                                 | 🚧      | ✔️                                                           | C                                                                |
+| Hiding process arguments from tests                                                       | ✔️       | ✔️                                                           | C                                                                |
 | Declaring that an individual test should fail                                             | 🚧      | ✖️                                                           | [Lib](http://hackage.haskell.org/package/tasty-expected-failure) |
 | Declaring that at least one in a test group should fail                                   | 🚧      | C                                                           | [Lib](http://hackage.haskell.org/package/tasty-expected-failure) |
-| Using scarce resources across tests                                                       | 🚧      | C                                                           | ?                                                                |
+| Using scarce resources across tests                                                       | 🚧      | C                                                           | C                                                                |
```

Comment:
Also possible with `withResource`, but there are no helper functions for it, just like there aren't in `hspec` either.